babel.config.js

```diff
@@ -1,12 +1,6 @@
-module.exports = {
-  presets: [
-    [
-      'module:metro-react-native-babel-preset',
-      { useTransformReactJSXExperimental: true },
-    ],
-  ],
-  plugins: [
-    ['@babel/plugin-transform-react-jsx', { runtime: 'automatic' }],
-    'react-native-reanimated/plugin',
```

Comment:
The Reanimated plugin is [automatically configured](https://docs.expo.dev/versions/latest/sdk/reanimated/#installation) in `babel-preset-expo`